packages/astro/src/runtime.ts

```diff
@@ -250,6 +250,7 @@ async function load(config: RuntimeConfig, rawPathname: string | undefined): Pro
     }
 
     if (err instanceof NotFoundError && rawPathname) {
+      console.log('ERR', err);
```

Comment:
Remove?

Reply:
This PR isn't really ready for review. I'll convert it to a Draft for now.